docker/image/console/root/lib/task/build/backend.sh.twig

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+function task_build_backend()
+{(
+    set -e
+    cd {{ @('backend.path') }}
+    
+    if [ ! {{ @('backend.build.when')|raw }} ]; then
+        return 0;
+    fi
+
+    {% for step in @('backend.build.steps') -%}
+    {{ step|raw }}
+    {% endfor %}
+
+    set +e
```

Comment:
No need to `set +e` when within a subshell `(...)`

Reply:
updated